### PR TITLE
feat(gas):implementation module whithout triggering

### LINKED ISSUE #147: Implement the GAS module

### PR DESCRIPTION
**Pull request by letypequividelespoubelles**

### CODE REVIEW

File: arithmetization/src/main/java/net/consensys/linea/zktracer/module/gas/GasOperation.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+import java.math.BigInteger;
+
+import net.consensys.linea.zktracer.container.ModuleOperation;
+import net.consensys.linea.zktracer.types.UnsignedByte;
+import org.apache.tuweni.bytes.Bytes;
+
+public class GasOperation extends ModuleOperation {
+
+  final short ctMax = 7;
```
Comment (letypequividelespoubelles) [inline arithmetization/src/main/java/net/consensys/linea/zktracer/module/gas/GasOperation.java]: constant from lisp !

File: arithmetization/src/main/java/net/consensys/linea/zktracer/module/gas/GasOperation.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  GasParameters gasParameters;
```
Comment (letypequividelespoubelles) [inline arithmetization/src/main/java/net/consensys/linea/zktracer/module/gas/GasOperation.java]: should have @EqualsAndHAsh annotations for the three arg, GAS is stateless

File: gradle/trace-files.gradle [hunk 3]
```diff
@@ -68,7 +68,7 @@ tasks.register('binreftable', TraceFilesTask) {
 //
 // Put here modules following the conventional MODULE/columns.lisp naming scheme.
 //
-['bin', 'blockhash', 'euc', 'mmio', 'shf', 'wcp', 'rlptxn', 'rom', 'romlex', 'trm', 'stp', 'loginfo', 'logdata'].each {moduleName ->
+['bin', 'blockhash', 'euc', 'mmio', 'shf', 'wcp', 'rlptxn', 'rom', 'romlex', 'trm', 'stp', 'loginfo', 'logdata', 'gas'].each {moduleName ->
```
Comment (letypequividelespoubelles) [inline gradle/trace-files.gradle]: should be moved to modules with constants, as you have to extract CTMAX from lisp

**Review by letypequividelespoubelles — COMMENTED** [review]
LGTM, we'll do the triggering in a separate PR

The @equalsAndHash business should be done

**Review by powerslider — APPROVED** [review]

**Pull request merged**